src/Service/ServiceManagerConfig.php

```diff
@@ -94,15 +95,20 @@ public function __construct(array $configuration = [])
     {
         $this->initializers = [
             'EventManagerAwareInitializer' => function ($instance, ServiceLocatorInterface $serviceLocator) {
-                if ($instance instanceof EventManagerAwareInterface) {
-                    $eventManager = $instance->getEventManager();
+                if (! $instance instanceof EventManagerAwareInterface) {
+                    return;
+                }
+
+                $eventManager = $instance->getEventManager();
```

Comment:
For those not following the whole thread, I answered this: `EventManagerAwareInterface` extends `EventManagerCapableInterface`, which defines the getter.